### PR TITLE
GEN-1404: Discounts view

### PR DESCRIPTION
**Pull request by sladan-hedvig**
- Updated discounts view

## Checklist

- [ ] Dark/light mode verification
- [ ] Landscape verification
- [ ] iPad verification
- [ ] iPod/iPhone 12 mini/iPhone SE verification

### CODE REVIEW

**Comment by sladan-hedvig** [comment]
Missing unreedeem campaign.

**Comment by sladan-hedvig** [comment]
I think it is ready now

File: Projects/Payment/Sources/Screens/PaymentDiscounts/AddCampaingCodeView.swift [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    var body: some View {
+        ZStack(alignment: .center) {
+            TextInputView(vm: vm.inputVm).opacity(vm.codeAdded ? 0 : 1)
+            SuccessScreen(title: L10n.paymentsDiscountAdded).opacity(vm.codeAdded ? 1 : 0)
+                .offset(y: -32)
+        }
+        .toolbar {
+            ToolbarItem(placement: .principal) {
+                hText(vm.codeAdded ? "" : L10n.paymentsAddCampaignCode)
+                //                if !vm.codeRemoved{
```
Comment (julia-andersson) [inline Projects/Payment/Sources/Screens/PaymentDiscounts/AddCampaingCodeView.swift:18]: remove?

Comment (julia-andersson) [reply Projects/Payment/Sources/Screens/PaymentDiscounts/AddCampaingCodeView.swift:18]: or maybe you want to use this later?

**Review by julia-andersson — APPROVED** [review]

File: Projects/Payment/Sources/Screens/PaymentDiscounts/AddCampaingCodeView.swift (hunk 1 above)
Comment (sladan-hedvig) [reply Projects/Payment/Sources/Screens/PaymentDiscounts/AddCampaingCodeView.swift]: Forgot about it. Fixed now :) tack!

**Pull request merged**